### PR TITLE
Move library out.

### PR DESCRIPTION
**Pull request by adam-vessey**
Library should now be installed separately.

### CODE REVIEW

**Comment by willtp87** [comment]
Dependent on: https://bitbucket.org/rjerome/citeproc-php/pull-request/2/basic-handling-of-csl-101s-et-al-use-last/diff Should likely reference a specific commit as blessed in the readme when/if rjerome merges.

**Comment by willtp87** [comment]
Ticket's been around waiting for input for some time: https://jira.duraspace.org/browse/ISLANDORA-1015

File: .travis.yml [hunk 1]
```diff
@@ -18,10 +18,12 @@ before_install:
   - git clone -b 7.x git://github.com/Islandora/islandora_xacml_editor.git
   - git clone -b 7.x git://github.com/Islandora/islandora_bookmark.git
   - git clone -b 7.x git://github.com/Islandora/islandora_batch.git
+  - hg clone https://bitbucket.org/rjerome/citeproc-php
```
Comment (adam-vessey) [inline .travis.yml]: Need to know where to grab this from, since we're to roll our own fork...

File: modules/citeproc/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Provides a Drupal interface to the [citeproc-php](https://bitbucket.org/rjerome/citeproc-php) library.
+
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [csl](https://github.com/Islandora/islandora_scholar/tree/7.x/modules/csl)
+
+Additionally, it is necessary to install the [citeproc-php](https://bitbucket.org/rjerome/citeproc-php)
```
Comment (adam-vessey) [inline modules/citeproc/README.md]: `citeproc-php` reference which will have to change to whatever fork.

File: modules/citeproc/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Citeproc
+
+## Introduction
+
+Provides a Drupal interface to the [citeproc-php](https://bitbucket.org/rjerome/citeproc-php) library.
```
Comment (adam-vessey) [inline modules/citeproc/README.md]: `citeproc-php` reference which will have to change to whatever fork.

**Comment by daniel-dgi** [comment]
Yah, that's been done for a while.  
https://github.com/Islandora/citeproc-php

Do I need to do anything else for you guys?

**Pull request merged**